app/lib/SolutionConceptTagger.coffee

```diff
@@ -1,10 +1,14 @@
 concepts = require 'schemas/concepts'
 
-module.exports = TagSolution = (solution) ->
-  code = solution.source
+module.exports = TagSolution = ({source, ast, language}) ->
```

Comment:
Can we rename `ast` here so it's not repurposed later?

Reply:
👍